src/components/com_tjcertificate/administrator/models/certificate.php

```diff
@@ -130,4 +130,25 @@ protected function populateState()
 		$id = ($jinput->get('id'))?$jinput->get('id'):$jinput->get('id');
 		$this->setState('certificate.id', $id);
 	}
+
+	/**
+	 * Method to get content HTML.
+	 *
+	 * This method provides the tjlms course/jt event info HTML.
+	 *
+	 * @param   int     $id      id 
+	 * @param   string  $client  client
+	 *
+	 * @since   __DEPLOY_VERSION__ 
+	 * 
+	 * @return  HTML
+	 */
+	public function getContentInfo($id,$client)
```

Comment:
- Rename this method to be more specific, say `getCertificateProviderInfo()`
- What is the `$id` for ? Maybe it should be `$content_id` ?